planning/behavior_velocity_planner/include/scene_module/stop_line/scene.hpp

```diff
@@ -85,8 +89,9 @@ class StopLineModule : public SceneModuleInterface
 
   geometry_msgs::msg::Point getCenterOfStopLine(const lanelet::ConstLineString3d & stop_line);
 
-  boost::optional<StopLineModule::SegmentIndexWithPoint2d> findCollision(
-    const autoware_auto_planning_msgs::msg::PathWithLaneId & path, const LineString2d & stop_line);
+  boost::optional<StopLineModule::SegmentIndexWithPoint2d> findCollisionAroundIndex(
+    const autoware_auto_planning_msgs::msg::PathWithLaneId & path, const LineString2d & stop_line,
+    const size_t index);
```

Comment:
What is the const size_t index? :pray: 
My assumption is that it will be an argument to a function like this
```
  boost::optional<StopLineModule::SegmentIndexWithPoint2d> findCollisionAroundIndex(
    const autoware_auto_planning_msgs::msg::PathWithLaneId & path, const LineString2d & stop_line,
    const size_t search_range_min_path_index, const size_t search_range_max_path_index);
```

```
  boost::optional<StopLineModule::SegmentIndexWithPoint2d> findCollisionAroundIndex(
    const autoware_auto_planning_msgs::msg::PathWithLaneId & path, const LineString2d & stop_line,
    const std::tuple<size_t, size_t> search_range_path_index);
```